### PR TITLE
[one-cmds] Add an option for FuseMulWithFullyConnectedPass

### PR DESCRIPTION
**Pull request by jiwaszki**
This commit adds one-cmd option for FuseMulWithFullyConnectedPass.

ONE-DCO-1.0-Signed-off-by: Jan Iwaszkiewicz <j.iwaszkiewi@samsung.com>

### CODE REVIEW

**Comment by jiwaszki** [comment]
* For https://github.com/Samsung/ONE/issues/13515
* From https://github.com/Samsung/ONE/pull/13528

File: compiler/one-cmds/how-to-use-one-commands.txt [hunk 1]
```diff
@@ -171,6 +171,7 @@ Current transformation options are
 - fuse_batchnorm_with_conv : This fuses BatchNorm operator to convolution operator
 - fuse_batchnorm_with_dwconv : This fuses BatchNorm operator to depthwise convolution operator
 - fuse_batchnorm_with_tconv : This fuses BatchNorm operator to transpose convolution operator
+- fuse_mul_with_fullyconnected: This fuses Mul operator with the preceding FullyConnected operator if possible
```
Comment (seanshpark) [inline compiler/one-cmds/how-to-use-one-commands.txt:174]: plz fix order

File: compiler/one-cmds/onelib/constant.py [hunk 2]
```diff
@@ -44,6 +44,7 @@ class CONSTANT:
         'fuse_batchnorm_with_dwconv',
         'fuse_batchnorm_with_tconv',
         'fuse_activation_function',
+        'fuse_mul_with_fullyconnected',
```
Comment (seanshpark) [inline compiler/one-cmds/onelib/constant.py:47]: plz fix order

File: compiler/one-cmds/onelib/constant.py [hunk 3]
```diff
@@ -122,6 +123,7 @@ class CONSTANT:
         ('fuse_batchnorm_with_conv', 'fuse BatchNorm op to Convolution op'),
         ('fuse_batchnorm_with_dwconv', 'fuse BatchNorm op to Depthwise Convolution op'),
         ('fuse_batchnorm_with_tconv', 'fuse BatchNorm op to Transposed Convolution op'),
+        ('fuse_mul_with_fullyconnected', 'fuse Mul op to FullyConnected op'),
```
Comment (seanshpark) [inline compiler/one-cmds/onelib/constant.py:126]: ditto

**Comment by seanshpark** [comment]
will check/land after all dependent PRs has landed.

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**